proposals/0272-sdl-javascript-manager-layer.md

```diff
@@ -853,23 +853,23 @@ class ChoiceSetManagerBase extends SubManagerBase {
     presentChoiceSet (choiceSet, mode, keyboardListener)
     
     /**
-     * Presents a keyboard on the Head unit to capture user input
-     * @param {String} initialText - The initial text that is used as a placeholder text. It might not work on some head units.
+     * Presents a keyboard on the head unit to capture user input
+     * @param {String} initialText - The initial text that is used as placeholder text. It might not work on some head units
      * @param {KeyboardProperties} customKeyboardConfig - the custom keyboard configuration to be used when the keyboard is displayed
      * @param {KeyboardListener} listener - A keyboard listener to capture user input
-     * @return {Number} - A unique id that can be used to cancel this keyboard. If `null`, no keyboard was created.
+     * @return {Number} - A unique ID that can be used to cancel this keyboard. If `null`, no keyboard was created
     */
     presentKeyboard (initialText, customKeyboardConfig, listener)
     
     /**
-     * Cancels the keyboard-only interface if it is currently showing. If the keyboard has not yet been sent to Core, it will not be sent.
-     * This will only dismiss an already presented keyboard if connected to head units running SDL 6.0+.
+     * Cancels the keyboard-only interface if it is currently showing. If the keyboard has not yet been sent to Core, it will not be sent
```

Comment:
```suggestion
     * Cancels the keyboard-only interface if it is currently showing. If the keyboard has not yet been sent to Core, it will not be sent.
```